### PR TITLE
New util functions named isValidRole and isEmptySig

### PR DESCRIPTION
**Pull request by jimni1222**
## Proposed changes

This PR introduces new util functions named isValidRole and isEmptySig.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/caver-js/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/caver-js)
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

refer #117 

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: packages/caver-utils/src/utils.js [hunk 1]
```diff
@@ -710,6 +710,35 @@ const isContractDeployment = (txObject) => {
...
+  switch(role) {
+    case 'transactionKey':
+    case 'updateKey':
+    case 'feePayerKey':
+      return true
+  }
+  return false
+}
+
+const isDefaultSig = (sig) => {
```
Comment (kjhman21) [inline packages/caver-utils/src/utils.js]: The function name is somewhat vague. Can we have another name for this function?

Comment (jimni1222) [reply packages/caver-utils/src/utils.js]: Hmmm how about isEmptySig or isEmptySignatures?

Comment (kjhman21) [reply packages/caver-utils/src/utils.js]: Yeah, anyone is good to me. :)

Comment (jimni1222) [reply packages/caver-utils/src/utils.js]: Function name is changed to isEmptySig :)

**Pull request merged**